src/WpBookingsUiModule.php

```diff
@@ -0,0 +1,344 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Data\Container\ContainerFactoryInterface;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\Modular\Module\AbstractBaseModule;
+
+class WpBookingsUiModule extends AbstractBaseModule
+{
+    private $bookingsPageId;
+
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param $key
+     * @param ContainerFactoryInterface $containerFactory The factory for creating container instances.
+     * @param $eventManager
+     * @param $eventFactory
+     * @throws \Dhii\Exception\InternalException
+     */
+    public function __construct($key, ContainerFactoryInterface $containerFactory, $eventManager, $eventFactory)
+    {
+        $this->_initModule(
+            $containerFactory,
+            $key,
+            [],
+            $this->_loadPhpConfigFile(WP_BOOKINGS_UI_MODULE_DIR . '/config.php')
+        );
+
+        $this->_initModuleEvents($eventManager, $eventFactory);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setup()
+    {
+        return $this->_createContainer([
+            'template_manager' => function () {
+                $templateManager = new TemplateManager($this->eventManager, $this->eventFactory);
+                $templateManager->registerTemplates($this->_getConfig()['templates']);
+                return $templateManager;
+            }
+        ]);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run(ContainerInterface $c = null)
+    {
+        /* @var EventManagerInterface $eventManager  */
+        $eventManager = $c->get('event_manager');
+
+        /** @var TemplateManager $templateManager */
+        $templateManager = $c->get('template_manager');
+
+        $assetsContainer = $this->_getContainerFactory()->make([
+            'definitions' => $this->_getConfig()['assets']

```

Comment:
This looks wrong, because it assumes that the config is an array. Entries should be accessed by key. If this requires extra implementation, use `_containerGet()` or `_containerGetPath()`.